controllers/providers/aws/aws.go

```diff
@@ -218,6 +219,20 @@ func GetInstanceFamily(instanceType string) string {
 	return ""
 }
 
+func GetInstanceArchitectures(typeInfo []*ec2.InstanceTypeInfo, instanceType string) []string {
+
+	instanceTypeInfo := GetInstanceTypeInfo(typeInfo, instanceType)
+	if instanceTypeInfo == nil || instanceTypeInfo.ProcessorInfo == nil {
+		return []string{}
+	}
+	var archs = []string{}
+	for _, s := range instanceTypeInfo.ProcessorInfo.SupportedArchitectures {
+		archs = append(archs, *s)
```

Comment:
Don't we need to dedup this list? if we go over multiple instances we might get same archs

Reply:
Also, are there any arcithectures besides `i386 | x86_64 | arm64` ?
We can hardcode those and avoid using this.
Not sure if we expect this list to change

Reply:
Oh nvm, it's for a single instance
Would be good if we can just use `aws.StringValue` instead of *s for nil safety (probably not a likely one though)